bucket/ncmpc.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "0.20",
+    "description": "Curses Music Player Daemon (MPD) client",
+    "homepage": "https://www.musicpd.org/clients/ncmpc",
+    "license": "GPL-2.0-or-later",
+    "suggest": {
+        "mpd": "mpd"
+    },
+    "url": "https://www.musicpd.org/download/ncmpc/0/ncmpc-0.20-win32.zip",
+    "hash": "ec7ba2ee6ed58a939eb0487e1e79169f695f7172d620790d734e6c3f9711fc1e",
+    "extract_dir": "ncmpc-0.20",
+    "bin": "ncmpc.exe",
```

```suggestion
    "bin": [
        [
            "ncmpc.exe",
            "ncmpc",
            "-f \"$dir\\config\" -k \"$dir\\keys\""
        ]
    ],
```

This will help users to modify their config by `(INSTALLATION DIR)\config`.
See [documentation](https://ncmpc.readthedocs.io/en/latest/) for details.

---

This is great! 👍